components/bl-signature-pad-component/src/utils/download.js

```diff
@@ -0,0 +1,16 @@
+import dataURLToBlob from './data-url-to-blob';
```

Comment:
You could create a `utils/file.js`

With functions that are related to working with files.

In this case import will be

`import {download, someOtherFunctionWithFile} from './utils/file'`